.github/scripts/shared_ptr_interface_check.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+source functions.sh
+
+FILE=../../homework/shared_ptr/shared_ptr.hpp
```

Comment:
I'm never 100% sure about using " in scripts, but I guess that it is safe to put paths in quotes.
```suggestion
FILE="../../homework/shared_ptr/shared_ptr.hpp"
```